src/Raven.Server/Documents/PeriodicExport/PeriodicExportRunner.cs

```diff
@@ -402,19 +406,8 @@ private void WriteStatus()
             using (_database.DocumentsStorage.ContextPool.AllocateOperationContext(out context))
             using (var tx = context.OpenWriteTransaction())
             {
-                var status = new DynamicJsonValue
-                {
-                    ["LastDocsEtag"] = _status.LastDocsEtag,
-                    ["LastExportAt"] = _status.LastExportAt.GetDefaultRavenFormat(),
-                    ["LastFullExportAt"] = _status.LastFullExportAt.GetDefaultRavenFormat(),
-                    ["LastFullExportDirectory"] = _status.LastFullExportDirectory
-                };
-                var readerObject = context.ReadObject(status, Constants.Documents.PeriodicExport.StatusKey, BlittableJsonDocumentBuilder.UsageMode.ToDisk);
-                var putResult = _database.DocumentsStorage.Put(context, Constants.Documents.PeriodicExport.StatusKey, null, readerObject);
+                _database.DocumentsStorage.SetDatabasePeriodicExportStatus(context, _status);
```

Comment:
No, that is what Karmel is doing, and will be part of that.
Let us focus on making the code, as is, work properly.